settings/acceptable_versions.go

```diff
@@ -0,0 +1,143 @@
+package settings
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/cmdshared"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/unascribed/FlexVer/go/flexver"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var acceptableVersionsCommand = &cobra.Command{
+	Use:     "acceptable-versions",
+	Short:   "Manage your pack's acceptable Minecraft versions. This must be a comma seperated list of Minecraft versions, e.g. 1.16.3,1.16.4,1.16.5",
+	Aliases: []string{"av"},
+	Args:    cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		modpack, err := core.LoadPack()
+		if err != nil {
+			// Check if it's a no such file or directory error
+			if os.IsNotExist(err) {
+				fmt.Println("No pack.toml file found, run 'packwiz init' to create one!")
+				os.Exit(1)
+			}
+			fmt.Printf("Error loading pack: %s\n", err)
+			os.Exit(1)
+		}
+		var currentVersions []string
+		// Convert the interface{} to a string slice
+		for _, v := range modpack.Options["acceptable-game-versions"].([]interface{}) {
```

Comment:
Does this still work if the `acceptable-game-versions` option doesn't exist? Go lets you iterate over `nil` slices, but I'm not sure if the cast will fail

Reply:
I'll have to double check if it does. I sense it might not, so I'll probably need to add a check for that.